src/components/CalendarCell/CalendarCell.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { Text } from '../Text';
+import { CALENDARCELL_DATA_HOOKS, CALENDAR_DATA_KEYS } from './dataHooks';
+import styles from './CalendarCell.st.css';
+
+export enum Times {
+  previousMonth = 'PreviousMonth',
+  previousDays = 'PreviousDays',
+  currentDay = 'CurrentDay',
+  nextMonth = 'NextMonth',
+}
+
+export interface CalendarCellProps {
+  time: string;
+  children?: any;
+  timeType?: Times;
+  isStretchAble?: boolean;
+}
+
+interface DefaultProps {
+  isStretchAble: boolean;
```

Comment:
No need for a capital `A` 😊 
I would even change it to just `stretchable`
```suggestion
  stretchable: boolean;
```